tests/bot/modules/dispute/messages.js

```diff
@@ -0,0 +1,108 @@
+const { expect } = require('chai');
```

Comment:
@danfercf1 hey Daniel, as I told you in another PR, new files should go in TypeScript, not JavaScript

Reply:
But it's a test file, and it will not be executed by the script of package.json if it's a typescript file

Reply:
test files can be written in typescript too

Reply:
Yes, but they are not supported by the project

Reply:
Let's make it so?

Reply:
Totally agree, but itsn't part of this task

Reply:
well, given no .ts tests exists in master branch, you cannot technically create a PR that separates that task completely. I mean, that task is needed only now that you're creating a new test

Reply:
Theres are not the @types for chai, mocha, sinon in order to implement the tests using typescript

Reply:
Hey guys, I also agree new files should be .ts, but if we need to some some extra stuff to make it work I consider we should merge this as javascript and then work on migrating it to ts in another issue